lib/puppet/provider/package/chocolatey.rb

```diff
@@ -84,12 +84,16 @@ def install
       args << '-y'
     end
 
-    args << @resource[:install_options]
-
     if @resource[:source]
       args << '-source' << @resource[:source]
     end
 
+    args << @resource[:install_options]
+
+    if Gem::Version.new(PuppetX::Chocolatey::ChocolateyCommon.choco_version) >= Gem::Version.new(PuppetX::Chocolatey::ChocolateyCommon::MINIMUM_SUPPORTED_CHOCO_VERSION_EXIT_CODES)
```

Comment:
Nit: can these be made variables? Super long line makes its hard to read the important part